### DISCUSSION TITLE
Admin: Date Field format

### DISCUSSION THREAD

**Discussion by LMSSonos** [OrchardCMS/OrchardCore#10379] [Q&A]
Is there a way to change the format for the date picker? The admin UI changes according the browser from english to german, but the datepicker has always teh format month/day/year - is there a option to change this?

![image](https://user-images.githubusercontent.com/19758451/135517060-bfa4b342-9ca6-4e72-8a41-d2475f67efd2.png)

**Comment by hishamco** [comment]
Do you tried to set the default locale into `de-DE`?

**Reply by LMSSonos** [reply]
Yes, that's already the case
![image](https://user-images.githubusercontent.com/19758451/135669603-ce8b10d0-bdf4-45b4-b201-c9b6bc4fc1d2.png)

**Reply by hishamco** [reply]
Could you please let me know the default date format in German?

**Reply by LMSSonos** [reply]
Sure, it's dd.MM.yyyy (HH:mm:ss) - DateTime.Now.ToString() => `01.10.2021 21:02:14`

**Reply by hishamco** [reply]
We need to use `datetime-local` if I'm not wrong, let me check after ~ 2 hours

**Reply by LMSSonos** [reply]
that is probably the server date, correct? then it's english because the os is configured as us.

**Reply by hishamco** [reply]
The localization middleware will take care about setting the proper culture

**Reply by LMSSonos** [reply]
Sorry that I ask, but does that mean that I can change something in the middleware (configure it) - i did not find anything in the docs - or does it mean that the middleware localizes the text but not the dates ?

**Reply by hishamco** [reply]
> does it mean that the middleware localizes the text but not the dates

No .. no, there's something wrong I need to figure out

**Reply by hishamco** [reply]
The html date picker is localized properly in some browsers version, so waiting for @Skrypt to reply on the related issue, then I can test the jQuery UI one
